drivers/bluetooth/hci/userchan.c

```diff
@@ -68,7 +68,7 @@ static bool uc_ready(void)
 {
 	struct pollfd pollfd = { .fd = uc_fd, .events = POLLIN };
 
-	return (poll(&pollfd, 1, 0) == 1);
+	return (poll(&pollfd, 1, K_NO_WAIT) == 1);
```

Comment:
This is calling a libc API for the native posix build (i.e. the host libc), so this change is wrong (it will break once the macro expands to something else than an integer).

Reply:
Thanks!  Mis-placed parentheses in the regular expression caused the conversion to affect `poll` instead of `k_poll`.  Fixed now.